bot/cogs/help.py

```diff
@@ -0,0 +1,234 @@
+from inspect import Parameter
+import textwrap
+import typing as t
+
+import discord
+from discord.ext import commands, menus
+import discord.utils
+
+
+class HelpSource(menus.ListPageSource):
+    """The Help menu."""
+
+    def __init__(
+            self,
+            signature: t.Callable,
+            filter_commands: t.Coroutine,
+            prefix: str,
+            author: discord.User,
+            cogs: dict
+    ) -> None:
+        """Create the menu."""
+        self.get_command_signature = signature
+        self.filter_commands = filter_commands
+        self.prefix = prefix
+        self.menu_author = author
+        super().__init__(
+            [(cog, cogs[cog]) for cog in sorted(
+                cogs,
+                key=lambda cog: cog.qualified_name if cog else "ZZ"
+            )
+            ],
+            per_page=1,
+        )
```

Comment:
I'm not really a fan of this, list comprehensions should be smaller (preferably 1-liners), we could do something like this instead:
```suggestion
        sorted_cogs = sorted(
            cogs,
            key=lambda cog: cog.qualified_name if cog else "ZZ"
        )
        super().__init__(
            [(cog, cogs[cog]) for cog in sorted_cogs],
            per_page=1,
        )
```

Reply:
As you wish